include/grpcpp/generic/generic_stub_impl.h

```diff
@@ -156,22 +164,54 @@ class GenericStub final {
  private:
   std::shared_ptr<grpc::ChannelInterface> channel_;
 
-  void UnaryCallInternal(grpc_impl::ClientContext* context,
-                         const grpc::string& method,
+  void UnaryCallInternal(ClientContext* context, const grpc::string& method,
                          const grpc::ByteBuffer* request,
                          grpc::ByteBuffer* response,
-                         std::function<void(grpc::Status)> on_completion);
+                         std::function<void(grpc::Status)> on_completion) {
+    internal::CallbackUnaryCall(
+        channel_.get(),
+        grpc::internal::RpcMethod(method.c_str(),
+                                  grpc::internal::RpcMethod::NORMAL_RPC),
+        context, request, response, std::move(on_completion));
+  }
 
-  void PrepareUnaryCallInternal(grpc_impl::ClientContext* context,
+  void PrepareUnaryCallInternal(ClientContext* context,
                                 const grpc::string& method,
                                 const grpc::ByteBuffer* request,
                                 grpc::ByteBuffer* response,
-                                grpc_impl::ClientUnaryReactor* reactor);
+                                grpc_impl::ClientUnaryReactor* reactor) {
```

Comment:
I guess we are already within grpc_impl namespace so ClientUnaryReactor * should suffice